code/Controllers/SilverStripeNavigator.php

```diff
@@ -43,8 +43,8 @@ public function getItems()
     {
         $items = array();
 
-        $classes = ClassInfo::subclassesFor('SilverStripe\\CMS\\Controllers\\SilverStripeNavigatorItem');
-        unset($classes['SilverStripe\\CMS\\Controllers\\SilverStripeNavigatorItem']);
+        $classes = ClassInfo::subclassesFor(SilverStripeNavigatorItem::class);
+        array_shift($classes);
```

Comment:
nice one, so it doesn't need to be aware of the case sensitivity work

Reply:
Yes that's how all the code should be.